docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -663,6 +663,21 @@
                 </tr>
 
                 <tr>
+                    <td class="property" id="task-drop-producer-errors">task.drop.producer.errors</td>
+                    <td class="default">false</td>
+                    <td class="description">
+                        If true, producer errors will be logged and ignored. If false, producer errors will be propagated
+                        upward, which will usually fail the container. Failing the container is a safety precaution to
```

Comment:
"upward, either to the task, or to the run loop, in which case it will fail the container."